lib/Pod/To/Text.pm6

```diff
@@ -93,6 +93,9 @@ sub declarator2text($pod) {
         when Sub {
             'sub ' ~ $_.name ~ signature2text($_.signature.params)
         }
+        when .HOW ~~ Metamodel::EnumHOW {
+            "enum $_.perl() { signature2text $_.enums.pairs } \n"
```

Comment:
Are you sure you don't mean the following, here?

```
enum ' ~ $_.perl ~ ' ' ~ signature2text($_.enums.pairs) ~ " \n"
```

Reply:
`"$_.perl()"` is fine. Method calls on interpolated variables are supported by Raku